JS/light-dark-mode.js

```diff
@@ -1,3 +1,8 @@
+
+if (localStorage.getItem('theme') === null && location.pathname === "/index.html") {
+  console.log('yes null')
```

Comment:
assuming this to be redundant code so you can discard

Reply:
done